<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="5" passed="3" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-02-25T07:30:12Z" name="Suite" finished-at="2020-02-25T07:30:38Z" duration-ms="26296">
    <groups>
    </groups>
    <test started-at="2020-02-25T07:30:12Z" name="Test" finished-at="2020-02-25T07:30:38Z" duration-ms="26296">
      <class name="com.apex.ecommerce.signon.test.SignonTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:12Z" name="setUp" finished-at="2020-02-24T23:30:20Z" duration-ms="7697" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithAllBlanks()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:20Z" name="testWithAllBlanks" finished-at="2020-02-24T23:30:21Z" duration-ms="1217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithAllBlanks -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:21Z" name="close" finished-at="2020-02-24T23:30:21Z" duration-ms="167" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:21Z" name="setUp" finished-at="2020-02-24T23:30:24Z" duration-ms="3477" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithBlankUserId()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:24Z" name="testWithBlankUserId" finished-at="2020-02-24T23:30:25Z" duration-ms="866" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithBlankUserId -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:25Z" name="close" finished-at="2020-02-24T23:30:26Z" duration-ms="198" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:26Z" name="setUp" finished-at="2020-02-24T23:30:28Z" duration-ms="2863" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithInvalidCharacters()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:28Z" name="testWithInvalidCharacters" finished-at="2020-02-24T23:30:29Z" duration-ms="1029" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.apex.ecommerce.signon.test.SignonTest.testWithInvalidCharacters(SignonTest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithInvalidCharacters -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:30Z" name="close" finished-at="2020-02-24T23:30:30Z" duration-ms="104" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:30Z" name="setUp" finished-at="2020-02-24T23:30:33Z" duration-ms="2803" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithSuccessCredentials()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:33Z" name="testWithSuccessCredentials" finished-at="2020-02-24T23:30:34Z" duration-ms="1061" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithSuccessCredentials -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:34Z" name="close" finished-at="2020-02-24T23:30:34Z" duration-ms="97" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:34Z" name="setUp" finished-at="2020-02-24T23:30:37Z" duration-ms="2587" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testWithWrong()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:37Z" name="testWithWrong" finished-at="2020-02-24T23:30:37Z" duration-ms="831" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [valid] but found [Invalid Username / Password!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [valid] but found [Invalid Username / Password!]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.apex.ecommerce.signon.test.SignonTest.testWithWrong(SignonTest.java:73)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testWithWrong -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.apex.ecommerce.signon.test.SignonTest@57abad67]" started-at="2020-02-24T23:30:38Z" name="close" finished-at="2020-02-24T23:30:38Z" duration-ms="149" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.apex.ecommerce.signon.test.SignonTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
